Cobbduceus 1.1/X, Base(d) Healers (THIS KILLS TACTICAL KILLING AND CLONING???)

ExcessiveUseOfCobblestone opened the pull request:
##  About The Pull Request

### Buffs C2 Healing
I should have leaned to more healing initially, especially since they're damaging. Goal here is to make them more enticing.

### Dilutes Starting Meds / Borg Hydro Microdosing

Perhaps this was the biggest issue in that your DEFAULTS carried way too much of the chem specifically designed for lower dosages so people were getting smacked with needless damage consistently.

You can still make your own "pure" samples I guess, but I think you'll like the current values.

Borgs can now switch between 5u/2u with altclick.

### Adds a C3 Diluent Specifically for Minor Brute/Burn

Wouldn't make much sense to dilute it with air! Granibitaluri is a new C3 that slowly tops off your damage by order of brute then burn. The catch is you must be at 10 damage or less for the healing to apply. ODs at 50.

This is the reagent all C2s are diluted with regardless of what they heal (meaning brute/burn chems will heal a tad better but only when you're about healed anyways).

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Buffed C2s to make them more viable as contenders to some of the more oddity avenues you guys were using.

Dilutes C2s to make the roundstart items viable on their own without chemistry (or at least a chem master). Same with Borg Microdosing option.

Added the C3 since it was suggested by zxaber to have a form of healing more minor damage without going through the lengths of surgery.

Sorry in advance silicon players, the solutions are still pure in your hypo!

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: Anarcho-Capitalistabital-by
add: In General, Buffs healing thoroughput of C2s. You can stop paying your cloners overtime now!
tweak: The roundstart C2 methods are now diluted with a new reagent, Granibitaluri.
tweak: The borghypo now can switch to "microdosing", which is 2u transfer via AltClick
add: Granibitaluri is a C3 (no downside!!!) that tops off your brute THEN burn damage if they are at or  below 10 damage.
/:cl:

Thank you guys for being so patient and providing constructive feedback! G-g-g-good thing coders "play" haha!

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ExcessiveUseOfCobblestone committed:
Dilutes C2s with New Diluent (C3), Buffs Healing

kill and cl-

ExcessiveUseOfCobblestone committed:
borg love and bruh moment

wesoda25 commented:
i will keep saying it until i cant say it anymore, c2s should all damage the same organ (liver makes most sense). Also those healing values still aren't good enough, you should make it -3.

ExcessiveUseOfCobblestone commented:
whoops forgot to push second commit

ExcessiveUseOfCobblestone committed:
reverts musiver

idk why I changed those

Ty-the-Smonk commented:
KILL AND CLONE DEATH IS TEMPORARY KILL AND CLONE

PKPenguin321 commented:
I think people have been asking for a top-off chem ever since trekchem was removed so this is pretty welcome

tralezab commented:
nobody has been asking for that name though, that is one terrible chem name

anconfuzedrock commented:
one hit with a toolbox and it won't heal so this changes nothing

tralezab reviewed (commented):
just a reminder to fix some pathing

dr-bee commented:
Id rather just eat something. Vitamin is significantly better as a healer and easier to find. This isnt really a solution.

ExcessiveUseOfCobblestone committed:
pathing

anconfuzedrock commented:
when do we make sleeping heal people

ExcessiveUseOfCobblestone commented:
> 
> 
> Id rather just eat something. Vitamin is significantly better as a healer and easier to find. This isnt really a solution.

 food healing for 1/1 brute and burn 50% for no reason just goes to show how flippantly people just tossed healing around.

Just having 10u vitamin gives you 25 iterations which roughly translates to about 12.5 HP of both brute AND burn. stupid tbh

Krysonism commented:
The name of the new chem is horrible. That is all, thanks.

cacogen commented:
i don undesa

ExcessiveUseOfCobblestone committed:
night coding sweetie

PKPenguin321 commented:
Cobbducobbyeus 1.1/C, Cobb(y) Cobbers (THIS COBBS COBTICAL COBBING AND CLOBBING?C?)

ExcessiveUseOfCobblestone commented:
did I mention cobby was the author or this PR?

Dawson1917 commented:
>The catch is you must be at 10 damage or less for the healing to apply. 

just add passive healing at this point. you don't need chems for accidentally touching a lightbulb.

ExcessiveUseOfCobblestone commented:
I made the healing after I added the diluent since I didn't want things to just look like 2u of a chem.

AdamElTablawy commented:
what use is a chemical specifically for healing pathetic amounts of damage

at that point eat food or drink milk or something unless you've got an issue with milk healing people now

FancyCyclops commented:
This doesn't nerf the usage of these chems in murder, one of the most obvious ways to use these.

Difrit commented:
Can we at least stop making the new chem names sound like someone having a stroke

Akrilla commented:
medical shapin up

nemvar commented:
Granibitaluri seems a tad weak. As in... it might as well do nothing. 
If you take 50 units of it, it heals a whole of 12.5 damage. Compare that to lets say... nutriment, which heals half a brute damage per tick on average and as such is 5 times as strong as granibitaluri.
Nobody in their right mind will EVER use a medicine with a healing amount that is that low. If it only heals tiny boo boos, it may aswell have a decent heal rate of one (WHOLE) damage per second otherwise it is literally outclassed by every single food reagent in the game.

nemvar commented on code/modules/reagents/chemistry/reagents/medicine_reagents.dm:
```diff
@@ -1234,3 +1234,24 @@
+/datum/reagent/medicine/granibitaluri/on_mob_life(mob/living/carbon/M) //
+	if(M.getBruteLoss() <= 10)
+		M.adjustBruteLoss(-0.1)
+	else if(M.getFireLoss() <= 10)
```
This won't heal fireloss if bruteloss is 0. Is that intentional?

nemvar commented on code/modules/reagents/reagent_containers/borghydro.dm:
```diff
@@ -152,6 +153,15 @@ Borg Hypospray
 	if(empty)
 		. += "<span class='warning'>It is currently empty! Allow some time for the internal synthesizer to produce more.</span>"
 
+/obj/item/reagent_containers/borghypo/AltClick(mob/living/user)
```
Call parent.

ExcessiveUseOfCobblestone commented:
It's just suppose to be a diluent. For patches this is just immersion fluff, for syringes/gels its an actual diluent.

Nutrient/Vitamin healing sounds like an issue I need to fix, but not this PR.

ExcessiveUseOfCobblestone committed:
Update borghydro.dm

shellspeed1 commented:
> 
> 
> It's just suppose to be a diluent. For patches this is just immersion fluff, for syringes/gels its an actual diluent.
> 
> Nutrient/Vitamin healing sounds like an issue I need to fix, but not this PR. That said, nothing stops you from taking all 3 simultaneously.

Consider that nutriment and vitamin can make you fat which can cause mood debuffs and massive slowdown if you take more than a few units. I don't think they are as strong as you think consider the side effects of using it for anything beyond healing some minor injuries.

ExcessiveUseOfCobblestone committed:
SLAYING Libital with FACTS and LOGIC

ExcessiveUseOfCobblestone committed:
Update medicine_reagents.dm

wesoda25 commented:
Idk why everyone is complaining, this is precisely the chem we wanted. However, Cobby you should bump up the heal rates to at least 0.25 for granitbitluri, anything below that is absurd. Also work on your names.

wesoda25 commented:
Also you should flesh out more chems like gran, turn them into base level healers... a first tier of sorts....

ExcessiveUseOfCobblestone commented:
I'm going to change the name of multiple healing to be a generic one rather than a conglom, but in a sep PR since I don't want name discussion clouding balance

ExcessiveUseOfCobblestone committed:
Merge branch 'master' into liberalital

optimumtact commented:
To be honest literally every source of healing in the game needs to be reviewed and most of them probably need to be culled

KetasDotepasto commented:
Please also remove the white medkits. The bruise packs and burn lotions provide instant healing, which makes wound tending surgery superfluous.

optimumtact commented:
I'd rather have them only heal damage above a 50 or 60% cap so they can still be used to treat small wounds in each dept

ExcessiveUseOfCobblestone commented:
Yeah once I swing back around to chemland I'll definitely be looking at those (unless someone wants to work on that now).

Right now though i'm shifting gears towards more ways to interact with patients as a doctor not bound by chemistry (looking at you tox/ox/organs)

ExcessiveUseOfCobblestone committed:
perf/conv buff

2x Oxygen healing for same amount of tox

reverts scaling on min tox taken when not healing.

ExcessiveUseOfCobblestone committed:
didn't save sorry

ExcessiveUseOfCobblestone committed:
same healing, different conversion.

adamkad1 commented:
just port baymed, or atleast port peridaxon, or other organ healing chem...

ExcessiveUseOfCobblestone commented:
no
